car_goal_training:
  ros__parameters:
    max_steps_exploration: 5000
    g: 5
    step_length: 1.0